### DISCUSSION TITLE
Using Natural Language Processing (NLP) for improving readability of astrological facts

### DISCUSSION THREAD

**Discussion by sengiv** [gen-so/Genso.Astrology#5] [Ideas]
suggested by @salilnavgire

**Comment by ApurvaDwivediPracto** [comment]
Hi!

I went thru a little of the repository. I would like to contribute to your project in any way possible.

**Reply by sengiv** [reply]
[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/S6S0ELZGR)
